README.md

```diff
@@ -39,7 +39,7 @@ The source content for the manual and notebook examples can be found in the [pyt
 
 In addition to the core pytket package there are pytket extension modules which allow pytket to interface with quantum devices and simulators. Some extensions also provide interoperability with other software libraries such as qiskit, cirq and pennylane.
 
-For a list of available pytket extensions see the [extensions index page](https://docs.quantinuum.com/tket/api-docs/extensions).
+For a list of available pytket extensions see the [extensions index page](https://docs.quantinuum.com/tket/api-docs/extensions.html).
```

Comment:
Think the fact that the `.html` suffix is now needed here indicates a minor regression since the docs site migrated to azure.

Reply:
Isn't this page built within this repo? Can we not just do a relative link to extensions.md?

Reply:
Good spot. See [cdd71ff](https://github.com/CQCL/tket/pull/1938/commits/cdd71ffadd87789f999bc9d00fec19b9ca3b0255)

In the sphinx pages we can use the relative links. For the README and stuff outside of the docs build we have to use the full URL.

Reply:
Didn't click for me that this is in the README and not in a docs page, but thanks for changing the other instances!